alacritty/src/display/window.rs

```diff
@@ -404,6 +404,9 @@ impl Window {
 
     pub fn set_ime_allowed(&self, allowed: bool) {
         self.window.set_ime_allowed(allowed);
+        if allowed {
+            self.window.set_ime_purpose(ImePurpose::Terminal);
```

Comment:
Does this have to be reset every time? Or can we just set it once at startup?

Reply:
I think it persists internally in winit.

Reply:
Would be nice to just set it at startup once.